charts/stable/lldap/questions.yaml

```diff
@@ -41,7 +41,7 @@ questions:
                                           required: true
                                           private: true
                                           default: ""
-                                          min: 8
+                                          minLength: 8
```

Comment:
```suggestion
                                          max_length: 8
```

Reply:
max length? lldap has a min length requirement of 8....